privacyidea/lib/error.py

```diff
@@ -48,6 +48,7 @@ class ERROR:
     AUTHENTICATE_TOKEN_EXPIRED = 4305
     AUTHENTICATE_MISSING_RIGHT = 4306
     CA = 503
+    RESOURCE_NOT_FOUND = 601
```

Comment:
Why did yo choose 601? Why not a 404?
Or did you want to leave our 4xx errors to the authentication process? (I do not know the number-spaces myself)

Reply:
Works for me.